lnwallet/channel.go

```diff
@@ -67,28 +67,28 @@ const (
 type channelState uint8
 
 const (
-	// channelPending indicates this channel is still going through the
+	// ChannelPending indicates this channel is still going through the
 	// funding workflow, and isn't yet open.
 	channelPending channelState = iota
 
-	// channelOpen represents an open, active channel capable of
+	// ChannelOpen represents an open, active channel capable of
 	// sending/receiving HTLCs.
 	channelOpen
 
-	// channelClosing represents a channel which is in the process of being
+	// ChannelClosing represents a channel which is in the process of being
 	// closed.
 	channelClosing
 
-	// channelClosed represents a channel which has been fully closed. Note
+	// ChannelClosed represents a channel which has been fully closed. Note
 	// that before a channel can be closed, ALL pending HTLCs must be
 	// settled/removed.
 	channelClosed
 
-	// channelDispute indicates that an un-cooperative closure has been
+	// ChannelDispute indicates that an un-cooperative closure has been
 	// detected within the channel.
 	channelDispute
 
-	// channelPendingPayment indicates that there a currently outstanding
+	// ChannelPendingPayment indicates that there a currently outstanding
```

Comment:
The `godoc` comments were changed, yet the variables themselves are the same. I'm guessing this was left over before reverting a prior change to make all of these public.